<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QompPluginGettunesDlg</class>
 <widget class="QDialog" name="QompPluginGettunesDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog { 
	color: black;
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #C1C7F5, stop: 1 #8ab4f6);
}

QTreeView, QListView, QListWidget { 
    background-color: #DFE1F5;
	border-radius: 5px;
	font: 14px;
}

QToolButton { 
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #f0f0f0, stop: 0.4 #ebebeb, stop: 0.7 #dedede, stop: 1.0 #d9d9d9); 
     border: 2px solid #799fd8; 
     border-top-left-radius: 5px; 
     border-bottom-left-radius: 5px; 
     border-bottom-right-radius: 5px; 
     border-top-right-radius: 5px; 
     width: 28px; 
     height: 28px; 
	 padding: 1px;
	 icon-size: 28px;
}

QToolButton:hover { 
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #f0f0f0, stop: 1 #DFE1F5);
	border: 2px solid #8ab4f6;
}

QToolButton:pressed { 
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #8faed5, stop: 0.4 #6dadfa, stop: 0.7 #368ef9, stop: 1.0 #358bf3); 
     border: 2px solid #698ebf; 
     border-top-left-radius: 5px; 
     border-bottom-left-radius: 5px; 
     border-bottom-right-radius: 5px; 
     border-top-right-radius: 4px; 
}

QToolButton:checked { 
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:0.191, y2:1, stop:0 rgba(223, 225, 245, 255), stop:0.39196 rgba(207, 209, 245, 255), stop:0.859296 rgba(109, 160, 219, 255), stop:1 rgba(62, 85, 221, 255));
     border: 2px solid #698ebf; 
     border-top-left-radius: 5px; 
     border-bottom-left-radius: 5px; 
     border-bottom-right-radius: 5px; 
     border-top-right-radius: 4px; 
}

QLabel { 
    color: #3244C9; 
}

QComboBox { 
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #f0f0f0, stop: 0.4 #ebebeb, stop: 0.7 #dedede, stop: 1.0 #d9d9d9);  
     border: 2px solid #799fd8;  
     border-radius: 5px;  
     padding-right: 5px; 
     padding-left: 5px; 
}

QComboBox::drop-down {
    width: 15px;
    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid;
    border-top-right-radius: 5px;
    border-bottom-right-radius: 5px;
}

QComboBox::down-arrow {
	icon-size: 8px;
	image: url(:/icons/arrow-down);
}

QPushButton { 
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #f0f0f0, stop: 0.4 #ebebeb, stop: 0.7 #dedede, stop: 1.0 #d9d9d9); 
     border: 1px solid #8d909c; 
     border-color: #8d909c; 
     border-radius: 5px;
	padding : 4px 8px;
}

QPushButton:pressed { 
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #aaaaaa, stop: 0.4 #d5d5d5, stop: 0.7 #d5d5d5, stop: 1.0 #c6c6c6); 
     border: 1px solid #717171; 
     border-color: #717171; 
}

QMenu {
    background-color: #8ab4f6;
    border: 1px solid #698ebf;
}

QMenu::item {
    background-color: transparent;
}

QMenu::item:selected {
	color: black;
   	background: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 0,
                                stop: 0 #DFE1F5, stop: 1 #8ab4f6);
}

QToolTip {
 	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #f0f0f0, stop: 0.4 	#ebebeb, stop: 0.7 #dedede, stop: 1.0 #d9d9d9);
	border-radius: 5px; 
	border: 2px solid #698ebf;
	padding: 2px;
}

QDialogButtonBox {
	dialogbuttonbox-buttons-have-icons: 1;
	dialog-ok-icon: url(&quot;:/icons/ok&quot;);
	dialog-cancel-icon: url(&quot;:/icons/cancel&quot;);
}

QTabWidget:pane {
	border-radius: 5px;
}

QTabBar::tab {
	border: 2px solid #698ebf;
    border-top-left-radius: 5px;
    border-top-right-radius: 5px;
	padding: 3px 6px;
	width: 100px;
}

QTabBar:tab-bar {
	left: 5px;
}

QTabBar::tab:hover {
 	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #C1C7F5, stop: 1 #8ab4f6);   
}

QTabBar::tab:selected {
	 background: qlineargradient(spread:pad, x1:0, y1:0, x2:0.191, y2:1, stop:0 			rgba(223, 225, 245, 255), stop:0.39196 rgba(207, 209, 245, 255), stop:0.859296 rgba(109, 160, 219, 255), stop:1 rgba(62, 85, 221, 255));
}

 QScrollBar:vertical {
     border: 2px solid #799fd8;
     background: #DFE1F5;
     width: 18px;
	 margin: 25px 0px;
 }
 QScrollBar::handle:vertical {
     min-height: 20px;
	 border-radius: 5px;
	 border: 2px solid #799fd8;
	 background: rgba(62, 85, 221, 255);
 }
 QScrollBar::add-line:vertical {
     border: 2px solid #799fd8;
     background: rgba(62, 85, 221, 255);
     height: 20px;
	 border-bottom-left-radius: 5px;
     border-bottom-right-radius: 5px;
     subcontrol-position: bottom;
     subcontrol-origin: margin;
	 image: url(:/icons/arrow-down);
	 margin-bottom: 5px;
 }

 QScrollBar::sub-line:vertical {
     border: 2px solid #799fd8;
	 border-top-left-radius: 5px;
     border-top-right-radius: 5px;
     background: rgba(62, 85, 221, 255);
     height: 20px;
	 subcontrol-position: top;
     subcontrol-origin: margin;
	 image: url(:/icons/arrow-up);
	 margin-top: 5px;   
 }

 QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
     background: none;
 }</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="mainLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="cb_search">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="maxCount">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_search">
         <property name="text">
          <string>Search</string>
         </property>
         <property name="icon">
          <iconset resource="../qomp.qrc">
           <normaloff>:/icons/search</normaloff>:/icons/search</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QompBusyLabel" name="lb_busy">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QompBusyLabel</class>
   <extends>QLabel</extends>
   <header>qompbusylabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qomp.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QompPluginGettunesDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QompPluginGettunesDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
